elfpy/utils/math/fixed_point.py

```diff
@@ -481,7 +481,8 @@ def ln(x: int) -> int:
         Reverts if value is negative or 0.
         """
         if x <= 0:
-            raise ValueError("FixedPointMath_NegativeOrZeroInput")
+            print(f"{x=}")
```

Comment:
No prints! I also fixed all of the fixed point error messages in one of my prs, I can add the value as well in that one as well. So can you revert these changes in yours?